charts/lm-logs/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v2
 description: A Helm chart for sending k8s logs to Logic Monitor
 name: lm-logs
-icon: https://logicmonitor.github.io/helm-charts/lm_logo.png
-version: 0.4.0
+version: 0.5.1
```

Comment:
Shouldn't it be `0.5.0`?

Reply:
0.5.0 remained unreleased and it was patch fixed for helm-charts-qa